### ISSUE TITLE
Missing header files.

### ISSUE THREAD

**Issue by abhijeeetkumar** [eric-rotenberg/CVP#6]
Hi,

It seems something very trivial is missing in cvp2v2.1 release. fixed the header file locally. If others are facing then please fix it in master.

make all
make -C lib DEBUG=0
make[1]: Entering directory `/home/mdl/amk7371/CVP-cvp2v2.1/lib'
g++ -std=c++11 -I.. -I../lib  -O3 -DGZSTREAM_NAMESPACE=gz -c -o cvp.o cvp.cc
In file included from cvp.cc:36:0:
uarchsim.h:28:17: fatal error: map.h: No such file or directory
 #include <map.h>
                 ^
compilation terminated.
make[1]: *** [cvp.o] Error 1
make[1]: Leaving directory `/home/mdl/amk7371/CVP-cvp2v2.1/lib'
make: *** [lib] Error 2

**Comment by aperais** [comment]
You are correct, the header was added back on the tip of the cvp2v2 branch, but it is not in the release. This header being pretty standard some compilers may go get it regardless (I assume, since we have not seen complaints).

The master branch should be fine, as the header was removed by mistake on the cvp2v2 branch by a previous commit. I will most likely make a final release with the fix although at this is not really blocking, as you pointed out.
